app/src/test/java/by/academy/jc/Zabrodski/ht2/DirtySumopenTest.java

```diff
@@ -0,0 +1,32 @@
+package by.academy.jc.Zabrodski.ht2;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class DirtySupomenTest {
```

Comment:
Looks like you provided two copies of `DirtySupomenTest` but in different packages. I think we can stay with one